app.js

```diff
@@ -23,8 +23,8 @@ app.use(frameguard({
 }));
 app.use(helmet.csp({
   directives:{
-    scriptSrc: ["'self'","'unsafe-inline'","data:", "https://cdn.optimizely.com", "https://app.optimizely.com", "https://basket.mozilla.org","https://*.shpg.org/", "https://www.google-analytics.com/"],
-    connectSrc:["'self'", "206878104.log.optimizely.com", "https://basket.mozilla.org/"],
+    scriptSrc: ["'self'","'unsafe-inline'","data:", "https://cdn.optimizely.com", "https://app.optimizely.com", "https://basket.mozilla.org", "https://basket-dev.allizom.org","https://*.shpg.org/", "https://www.google-analytics.com/"],
+    connectSrc:["'self'", "206878104.log.optimizely.com", "https://basket.mozilla.org/", "https://basket-dev.allizom.org"],
```

Comment:
I wanted to add these so local dev and staging can test submitting to make sure proper params are being passed in and that it succeeds.